edx_proctoring/api.py

```diff
@@ -640,9 +642,78 @@ def update_attempt_status(exam_id, user_id, to_status, raise_if_not_found=True,
         exam_attempt_obj.started_at = datetime.now(pytz.UTC)
         exam_attempt_obj.save()
 
+    if ProctoredExamStudentAttemptStatus.needs_status_change_email(exam_attempt_obj.status):
```

Comment:
We should not send a status email for "practice exams" as well as exams which were not taken as proctored. Both of these have corresponding flags/fields in the Exam and Attempt models. Can you also add a unit test to make sure practice exams and tests taken non-proctored don't send email. Thanks.